app/views/services/index.html.erb

```diff
@@ -30,10 +30,11 @@
             <%= f.label :service_name, class: "govuk-label govuk-label--m" %>
             <% f.object.errors.each do |error|  %>
               <p class="govuk-error-message"><%= error.message %></p>
+              <div id="hint">Give your form a name, your form name cannot begin with a number</div>
             <% end %>
-            <%= f.text_field :service_name, class: "govuk-input" %>
+            <div class="govuk-hint" id="hint">Give your form a name, your form name cannot begin with a number</div>
+            <%= f.text_field :service_name, class: "govuk-input", 'aria-describedby' => 'hint'%>
```

Comment:
While it's not necessarily wrong, I would use a more specific `id` than 'hint'.  `id` attributes must be unique to the whole page. Not actually an issue on this page, but its best to be a bit more specific to avoid conflicts.  Usually combining the field name and 'hint' is enough e.g. 'service-name-hint'

The actual text content of the hint should be moved to the translations file (config/locales/en.yml).